nbproject/private/private.properties

```diff
@@ -0,0 +1 @@
+browser=Chrome.INTEGRATED
```

Comment:
Seems that this file and the other Netbeans project configuration files got in the PR by mistake. Delete them to keep the repo clean